crates/rolldown/tests/esbuild/loader/loader_file_common_js_and_es6/artifacts.snap

```diff
@@ -13,16 +13,13 @@ var y_default = "y";
 
 //#endregion
 //#region x.txt
-var x_exports = {};
-__export(x_exports, { default: () => x_default });
-var x_default;
-var init_x = __esm({ "x.txt"() {
-	x_default = "x";
+var require_x = __commonJS({ "x.txt"(exports, module) {
+	module.exports = "x";
 } });
 
 //#endregion
 //#region entry.js
-const x_url = (init_x(), __toCommonJS(x_exports));
+const x_url = require_x();
```

Comment:
This is expected, there are still some diff between esbuild, because we don't support `file` loader yet, https://hyrious.me/esbuild-repl/?version=0.23.0&b=e%00entry.js%00const+x_url+%3D+require%28%27.%2Fx.txt%27%29%0Aimport+y_url+from+%27.%2Fy.txt%27%0Aconsole.log%28x_url%2C+y_url%29%0A&b=%00x.txt%00x&b=%00y.txt%00y&o=%7B%0A++treeShaking%3A+true%2C%0A++external%3A+%5B%22c%22%2C+%22a%22%2C+%22b%22%5D%2C%0A%22bundle%22%3A+true%2C%0Aformat%3A+%22esm%22%0A%7D